app/src/components/SearchBar.jsx

```diff
@@ -2,17 +2,23 @@ import React, { useState } from "react";
 import "../stylesheets/searchBar.scss";
 import closeSvg from '../assets/close.svg'
```

Comment:
camel or pascal case for image files? use consistent naming conventions, preferably across the whole project but at least within the same file